### DISCUSSION TITLE
Action chaining

### DISCUSSION THREAD

**Discussion by aman-kaur90** [hasura/graphql-engine#9326] [Question]
I have created two action(with different rest API) and now I want use response of first action as request of other.
Is it possible??

**Comment by adas98012** [comment]
@aman-kaur90 
Can you add more details about your use case ?  We can then provide you some solutions.

**Reply by aman-kaur90** [reply]
Hi @adas98012 ,
I have one graphql hasura server where I have created 2 actions.
Action1 <api 1.1>:- fetchemailId(GET) -> Response{ emailId ="xyz@gmail.com"}
Action2 <api 2.1>: fetchemailStatus(POST)-> Request{ emailId ="xyz@gmail.com"} -> Response{isVerified=true}

And both this action working perfectly separately and client consume response for both actions seperatly.
But I want Action1 and Action 2 to be linked in which action 1 response will automatically transferred as request to action 2 and client will consume action 2 response only

**Reply by adas98012** [reply]
@aman-kaur90 
Thanks for providing the details. Unfortunately, it is not possible to pipe the output of one action to the input of a second action, with the current Hasura GraphQL engine. You would have to write code to call both the actions (in series), passing result of one action into the other.
